index.html

```diff
@@ -7,33 +7,60 @@
         <script type="text/javascript" src="./js/changeIcon.js" defer></script>
     </head>
     <body>
-        <header>
-            <div class="logo">
-                <a href="index.html">
-                    <img src="./svg/logo.svg" alt="Logo"/>
-                    <p>Apartment Owners Association</p>
-                </a>
-            </div>
-            <div class="navigation" id="navButtons">
-                <a href="#about">About us</a>
-                <a href="#membership">Membership</a>
-                <a href="#services">Services</a>
-                <a href="#news">News and events</a>
-                <a href="#contact">Contact</a>
-            </div>
-            <div class="mobileIconContainer" onclick="handleClickMobile(this)">
-                <div class="mobileNavBar1"></div>
-                <div class="mobileNavBar2"></div>
-                <div class="mobileNavBar3"></div>
-            </div>
-        </header>
-        <section class="hero">
-            <div class="container">
+        <div class="page">
+            <header>
+                <div class="logo">
+                    <a href="index.html">
+                        <img src="./svg/logo.svg" alt="Logo"/>
+                        <p>Apartment Owners Association</p>
+                    </a>
+                </div>
+                <div class="navigation" id="navButtons">
+                    <a href="#about">About us</a>
+                    <a href="#membership">Membership</a>
+                    <a href="#services">Services</a>
+                    <a href="#news">News and events</a>
+                    <a href="#contact">Contact</a>
+                </div>
+                <div class="mobileIconContainer" onclick="handleClickMobile(this)">
+                    <div class="mobileNavBar1"></div>
+                    <div class="mobileNavBar2"></div>
+                    <div class="mobileNavBar3"></div>
+                </div>
+            </header>
+            <section class="hero">
+                <div class="container">
                 <span>
                     <h1>Apartment ownership with no headache</h1>
                     <sub>Wide selection of lorem usce volutpat lectus justo, ut suscipit felis congue ut. Vivamus ut ultricies ante dictum. </sub>
                 </span>
                 </div>
-        </section>
+            </section>
+            <section class="content">
+                <div class="container">
+                    <img src="images/Rectangle1.png"  alt="Tenant screening service"/>
```

Comment:
Attributes width and height are required for an image tag to prevent "layout shift" issues.